### PR TITLE
[android] Updated Android changelog for 8.4.0 stable release (ristretto)

### PR DESCRIPTION
**Pull request by langsmith**
This pr updates the Android changelog as part of the `8.4.0` (`ristretto`) release of the Maps SDK for Android.

### CODE REVIEW

File: platform/android/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
 ## master
 
+## 8.4.0 - September 25, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.4.0-beta.1...android-v8.4.0) since [Mapbox Maps SDK for Android v8.4.0-beta.1](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.4.0-beta.1): 
+
+### Bug fixes
+ - Fixing a crash when `CameraPosition` padding is set when using `MapSnapshotter`. [#15699](https://github.com/mapbox/mapbox-gl-native/pull/15699)
```
Comment (tobrun) [inline platform/android/CHANGELOG.md]: this is not in release-ristretto branch?

File: platform/android/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,15 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
...
+
+### Other Changes
+ - Bumped Java SDK dependency to stable 4.9.0 [#15700](https://github.com/mapbox/mapbox-gl-native/pull/15700)
+ - Migrate map feedback to GL JS–powered feedback form [#15623](https://github.com/mapbox/mapbox-gl-native/pull/15623)
```
Comment (tobrun) [inline platform/android/CHANGELOG.md]: this is not in release-ristretto branch?

Comment (langsmith) [reply platform/android/CHANGELOG.md]: Cherry-picked to `ristretto` in https://github.com/mapbox/mapbox-gl-native/pull/15714

File: platform/android/CHANGELOG.md (hunk 1 above)
Comment (langsmith) [reply platform/android/CHANGELOG.md]: Cherry-picked to `ristretto` in https://github.com/mapbox/mapbox-gl-native/pull/15714

**Comment by langsmith** [comment]
⚠️ Don't merge until https://github.com/mapbox/mapbox-gl-native/pull/15714 lands ⚠️

**Pull request merged**